_meta/beat.yml

```diff
@@ -1009,6 +1009,12 @@ output.elasticsearch:
 # Set to true to log messages in json format.
 #logging.json: false
 
+# Set to true, to log messages with minimal required Elastic Common Schema (ECS)
+# information. Recommended to use in combination with `logging.json=true`
```

Comment:
```suggestion
# It is recommended to use set `logging.json=true` when enabling ECS logging
```

Reply:
Suggested changes for the suggested change
```suggestion
# It is recommended to set `logging.json=true` when enabling ECS logging.
```